support-frontend/app/controllers/Application.scala

```diff
@@ -258,11 +258,20 @@ class Application(
   }
 
   def contributeGeoRedirect(campaignCode: String): Action[AnyContent] = GeoTargetedCachedAction() { implicit request =>
-    val url = List(getGeoRedirectUrl(request.geoData.countryGroup, "contribute"), campaignCode)
+    val url = getGeoPath(request, campaignCode, "contribute")
+    RedirectWithEncodedQueryString(url, request.queryString, status = FOUND)
+  }
+
+  def guardianLightGeoRedirect(campaignCode: String): Action[AnyContent] = GeoTargetedCachedAction() {
```

Comment:
Why do we support the campaignCode? What is this? It's always `= ""` in the routes file?

Reply:
Its not need in ths redirect so have removed.